API.md

```diff
@@ -378,20 +393,32 @@ typically best not to shut down siad until files are no longer being repaired.
 
 `TimeRemaining` indicates how many blocks the file will be available for.
 
-#### /renter/file/delete
+#### /renter/files/load
 
-Function: Deletes a renter file entry. Does not delete any downloads or
-original files, only the entry in the renter.
+Function: Load a '.sia' into the renter.
 
 Parameters:
 ```
-nickname string
+filename string
 ```
-`nickname` is the nickname of the file that has been uploaded to the network.
+`filename` is the filepath of the '.sia' that is being loaded.
 
-Response: standard
+Response: standard.
+
+#### /renter/files/loadascii
 
-#### /renter/file/rename
+Function: Load a '.sia' into the renter.
+
+Parameters:
+```
+file string
+```
+`file` is the ascii representation of the '.sia' file being loaded into the
+renter.
```

Comment:
URL length limit restrictions may be an issue here. We may need to use a POST instead of a GET.